From f72e9af1724f78b30ab380fd09555438d40c72a7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lothar=20Wa=C3=9Fmann?= <LW@KARO-electronics.de>
Date: Mon, 18 Mar 2019 15:56:18 +0100
Subject: [PATCH 6/8] ARM64: dts: imx8mm-tx8m-1610: disable arm-idle

The arm-idle driver stalls the system upon reboot. Disable it for now
to be able to reboot.
---
 arch/arm64/boot/dts/freescale/imx8mm-tx8m-1610.dts | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/imx8mm-tx8m-1610.dts b/arch/arm64/boot/dts/freescale/imx8mm-tx8m-1610.dts
index 0b856b2ded19..0723705da28e 100644
--- a/arch/arm64/boot/dts/freescale/imx8mm-tx8m-1610.dts
+++ b/arch/arm64/boot/dts/freescale/imx8mm-tx8m-1610.dts
@@ -63,6 +63,19 @@
 
 &A53_0 {
 	arm-supply = <&buck2_reg>;
+	/delete-property/ cpu-idle-states;
+};
+
+&A53_1 {
+	/delete-property/ cpu-idle-states;
+};
+
+&A53_2 {
+	/delete-property/ cpu-idle-states;
+};
+
+&A53_3 {
+	/delete-property/ cpu-idle-states;
 };
 
 &fec1 {
-- 
2.20.1

